test/swiss_schema_test.exs

```diff
@@ -35,6 +35,8 @@ defmodule SwissSchemaTest do
 
   describe "use SwissSchema" do
     test "requires a :repo option" do
```

Comment:
This case should be renamed. With this PR, SwissSchema no longer requires a `:repo` opt; it accepts a `:repo` opt – which can be made optional if an app env `:default_repo` is provided.

Perhaps we should set up new test cases to test the configuration requirements. I think that maybe we should raise a custom error with setup instructions in case users misuse it (neither configuring the app env through config files nor providing an inline `:repo` opt).